### PR TITLE
6.1 metafiles

### PR DESCRIPTION
**Pull request by pedrobaeza**

### CODE REVIEW

**Comment by Daniel-CA** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+language: python
+
+python:
+  - "2.7"
+
+env:
+  - VERSION="6.1" ODOO_REPO="odoo/odoo"
+  - VERSION="6.1" ODOO_REPO="OCA/OCB"
```
Comment (oihane) [inline .travis.yml]: OCA/OCB does not have version 6.1 in github

Comment (pedrobaeza) [inline .travis.yml]: Fixed, thanks.

**Comment by oihane** [comment]
PEP8 fails, but LGTM :+1:

**Pull request merged**